pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -529,6 +554,47 @@ func WriteBackupLock(
 	return cloud.WriteFile(ctx, defaultStore, lockFileName, bytes.NewReader([]byte("lock")))
 }
 
+// WriteFilesListMetadataWithSSTs writes a "slim" version of manifest
+// to `exportStore`. This version has the alloc heavy `Files` repeated field
+// nil'ed out, and written to an accompanying SST instead.
+func WriteFilesListMetadataWithSSTs(
+	ctx context.Context,
+	exportStore cloud.ExternalStorage,
+	encryption *jobspb.BackupEncryptionOptions,
+	kmsEnv cloud.KMSEnv,
+	manifest *backuppb.BackupManifest,
+) error {
+	if err := writeFilesListMetadata(ctx, exportStore, backupbase.BackupMetadataName, encryption,
+		kmsEnv, manifest); err != nil {
+		return errors.Wrap(err, "failed to write the slim backup manifest")
```

Comment:
If you end up needing another CI for some reason, this error still has the word "slim" in it